plugins/Installation/lang/en.json

```diff
@@ -19,145 +19,145 @@
+        "NoConfigFileFound": "Could not find the Matomo configuration file, and you are trying to access a Matomo page.",
         "YouMayInstallPiwikNow": "You may %1$sinstall Matomo now%2$s",
-        "IfPiwikInstalledBeforeTablesCanBeKept": "If you installed Matomo before and have some tables in your DB, don't worry, you can reuse the same tables and keep your existing data!",
+        "IfPiwikInstalledBeforeTablesCanBeKept": "If you installed Matomo before and have some tables in your database, don't worry, you can reuse the same tables and keep your existing data.",
         "Optional": "Optional",
         "Password": "Password",
-        "PasswordDoNotMatch": "password do not match",
+        "PasswordDoNotMatch": "mismatching passwords",
         "PasswordRepeat": "Password (repeat)",
         "PercentDone": "%s %% Done",
-        "ProfessionalServicesAdTitle": "Advanced Analytics & Services",
-        "ProfessionalServicesfessionalServicesAdText": "Our network of experts can help your organization unlock the full power of Matomo analytics.",
+        "ProfessionalServicesAdTitle": "Advanced Analytics and Services",
+        "ProfessionalServicesfessionalServicesAdText": "Our network of experts can help your organization use Matomo analytics to its full potential.",
         "ProfessionalServicesfessionalDiscoverHow": "Learn about %1$sadditional products and services%2$s for Matomo and how to increase your conversions and revenue.",
         "PleaseFixTheFollowingErrors": "Please fix the following errors",
         "DefaultSettings": "Default Matomo settings",
         "DefaultSettingsHelp": "Matomo comes with default settings. You can customize them now or do so later in the admin screen.",
         "Requirements": "Matomo Requirements",
         "RestartWebServer": "After making this change, restart your web server.",
         "ReusingTables": "Reusing the Tables",
-        "PiwikOrgNewsletter": "email me with major Matomo community updates",
-        "ProfessionalServicesNewsletter": "send me information on %1$sProfessional services and products%2$s for Matomo",
-        "SeeBelowForMoreInfo": "See below for more information.",
-        "SetupWebsite": "Setup a Website",
-        "SetupWebsiteError": "There was an error when adding the website",
+        "PiwikOrgNewsletter": "email me about major Matomo community updates",
+        "ProfessionalServicesNewsletter": "send me info on %1$sProfessional services and products%2$s for Matomo",
+        "SeeBelowForMoreInfo": "More info available below.",
+        "SetupWebsite": "Set up a Website",
+        "SetupWebsiteError": "Could not add the website",
         "SetupWebSiteName": "Website name",
-        "SetupWebsiteSetupSuccess": "Website %s created successfully!",
+        "SetupWebsiteSetupSuccess": "%s website created",
```

Comment:
Suggest reverting this one as existing is clear

Reply:
"successfully" is redundant. It suggest a website can be created and communicated as such, but "unsuccessful" at that.
Exclamation marks, strange structure, just all bad.